File: BUSCAR-VALOR.js
const array_numeros=[3, 2, 6, 8, 7, 6, 6, 1, 4]

let numero_ingresado = parseInt(prompt("ingrese un numero:"))


function buscarValor(lista, valor){
   let numero_encontrado=lista.indexOf(valor) /* utilizando el método de los array indexOf() busco el número ingresado por su índice dentro del array, si hay varios valores iguales muestra el indice del primero encontrado */
   if (numero_encontrado !==-1){
        return numero_encontrado
    }
    else{
        return -1
    }

}

let lugar_array = buscarValor(array_numeros, numero_ingresado)

if (lugar_array !=-1){
    
    console.log("Valor ingresado:", numero_ingresado, "Posición:", lugar_array)   
}
else{
    console.log("El número ingresado no se encuentra en el array")
}

File: buscar-valor.py

numero_ingresado = int(input("Ingrese un valor: "))

lista_numeros = [3, 2, 6, 8, 7, 6, 6, 1,4]


def buscar_valor(lista, valor):
    if valor in lista:
        numero_encontrado=lista.index(valor) #index(): método que busca un elemento por su indice  dentro de una lista
        return numero_encontrado
    else:
        return -1

lugar_lista = buscar_valor(lista_numeros, numero_ingresado)

if lugar_lista != -1:
    print ("Valor ingresado:", numero_ingresado, " posición:", lugar_lista)
else:
    print ("El número ingresado no se encuentra en la lista.")


